lib/mumuki/laboratory/locales/es.yml

```diff
@@ -15,14 +15,15 @@ es:
   appendix: Apéndice
   appendix_teaser: ¿Querés saber más? <a href="%{link}">Consultá el apéndice de este capítulo</a>
   approved_message: Validado por mentor
-  are_you_sure: '¿Estás seguro que querés %{action}?'
-  ask_a_question: ¡Hacé una pregunta!
   approved_messages:
     one: validado
     other: validados
+  are_you_sure: '¿Estás seguro que querés %{action}?'
```

Comment:
Same comment as before. We should paraphrase this as _¿Confirmás que querés ....?_ or  something like that.

Reply:
Yes, that would be nice!